package module3;

public class TestExceptions {

	public static void main(String[] args) throws Exception {

//		Complex testcomp1 = new Complex(0, 0);
//		Complex testcomp2 = new Complex(2, 2);
//		
//		System.out.println(testcomp1.normalised());
//		System.out.println(Complex.divide(testcomp2, testcomp1));
		
//		ThreeVector testvec1 = new ThreeVector(0,0,0);
//		ThreeVector testvec2 = new ThreeVector(1,1,1);
//		
//		System.out.println(testvec1.unitVector());
//		System.out.println(ThreeVector.angle(testvec1, testvec2));
		
//		FallingParticle testpart1 = new FallingParticle(0,1);
//		FallingParticle testpart2 = new FallingParticle(1,-1);
//		FallingParticle testpart3 = new FallingParticle(1,1);
//		
//		FallingParticle.setH(5);
//		testpart3.drop(0);
//		System.out.println(testpart3.getT());
		
	}

}
